test/stability-tests/src/it/java/software/amazon/awssdk/stability/tests/TestRunner.java

```diff
@@ -21,12 +21,12 @@
     public static void main(String... args) {
         // You can add the tests you want to run here.
         // eg:
-//        try {
-//            S3AsyncStabilityTest s3AsyncStabilityTest = new S3AsyncStabilityTest();
-//            S3AsyncStabilityTest.setup();
-//            s3AsyncStabilityTest.putObject_getObject();
-//        } finally {
-//            S3AsyncStabilityTest.cleanup();
-//        }
+        //        try {
```

Comment:
?

Reply:
Those are for testing purpose. I'll move the code snippets to javadoc section.